pages/papers/active-record-migrations-history-of-creation.mdx

```diff
@@ -4,36 +4,32 @@ lang: en
 description: The history told with migrations.
 ---
 
-<div css={{ color: "red", fontStyle: "italic", fontSize: "2em" }}>
-  In Progress
-</div>
-
 **Tags:** _Ruby_, _Ruby on Rails_, _Migrations_, _Active Record_
 
-**Audience Level:** _~~Beginer~~_ _~~Intermediate~~_ _All_
+**Audience Level:** _Beginer, All_
 
 ### Elevator Pitch (300 characters)
 
 <div css={{ color: "#888", fontStyle: "italic", fontSize: "0.8em" }}>
 You have 300 characters to sell your talk. This is known as the "elevator pitch". Make it as exciting and enticing as possible.
 </div>
 
-Active Record Migrations es lo que mas me impacto de Ruby on Rails, como ayuda esto a versionar nuestra estructura de base de datos es lo mejor, para demostrar lo sencillo y hermoso que es escribir migraciones, me anime a contarles una historia con migraciones.
+Active Record Migrations is what most impact me Ruby on Rails, how this helps to version our database structure is the best, to show how simple and beautiful it is to write migrations, I encourage you to tell a story with migrations.
```

Comment:
Active Record Migrations is what had the biggest impact on me from Ruby on Rails; the way this helps to version our database structure is the best. To [...], I thought about telling you a story with migrations. **